pages/my-data/index.js

```diff
@@ -26,6 +26,13 @@ export default function MyData({ username, onLogin }) {
             Reservierungen &nbsp;
             <ArrowUpRightIcon />
           </StyledLink>
+          <StyledLink
+            aria-label="Zur Meine-Kommentare-Seite"
```

Comment:
Not "Zur", but "Zu" 
Even more beautiful I think: "Link zu meiner Kommentar-Seite"